keyfreq.sh

```diff
@@ -1,32 +1,39 @@
 #!/bin/bash
 
 
-# logs the key press frequency over 9 second window. Logs are written 
+# logs the key press frequency over 9 second window. Logs are written
 # in logs/keyfreqX.txt every 9 seconds, where X is unix timestamp of 7am of the
 # recording day.
 
 LANG=en_US.utf8
 
-helperfile="logs/keyfreqraw.txt" # temporary helper file
+helperfile='/dev/shm/keyfreqraw.txt'
```

Comment:
`/dev/shm` is almost always a `tmpfs` which is important for:

> Never store highly sensitive information like keystrokes (which potentially can contain passwords) on persistent storage.

Ref: https://superuser.com/questions/45342/when-should-i-use-dev-shm-and-when-should-i-use-tmp